### PR TITLE
feat(scrap.tf): added small and large image keys

### PR DESCRIPTION
**Pull request by Ali-Joon**
**Describe the changes in this pull request:**
Profile Pictures now appear as `largeImageKey`
Also various trading pages now have the `smallImageKey`
<details>
<summary> Proof showing the creation/modification is working as expected </summary>
<br>

sc1
sc2
</details>

### CODE REVIEW

**Comment by Ali-Joon** [comment]
I've also checked that Image URLs are not longer than 256 characters

**Comment by ndbiaw** [comment]
> I've also checked that Image URLs are not longer than 256 characters


sc1 

sc2

File: websites/S/scrap.tf/dist/metadata.json [hunk 1]
```diff
@@ -9,9 +9,9 @@
 		"en": "ScrapTF is the largest bot-based TF2 item trading, raffle, and auction website. Buy and sell hats, keys, unusuals, stranges, skins, and more."
 	},
 	"url": "scrap.tf",
-	"version": "1.0.3",
+	"version": "2.0.0",
```
Comment (Slowlife01) [inline websites/S/scrap.tf/dist/metadata.json:12]:
```suggestion
	"version": "1.1.0",
```

File: websites/S/scrap.tf/presence.ts [hunk 2]
```diff
@@ -71,6 +72,11 @@ presence.on("UpdateData", async () => {
 			);
 		presenceData.details = "Visiting a profile";
 		presenceData.state = `[${perm.textContent}] ${title.textContent}`;
+		presenceData.largeImageKey = (
+			document.querySelector(
+				"#dynamic-height-slate > div > div > div.col-md-9 > div > div > div.profile-upper > div.rev-banking-background.profile-background > div > div.profile-avatar.col-md-3 > div > img"
+			) as HTMLImageElement
+		).src;
```
Comment (Slowlife01) [inline websites/S/scrap.tf/presence.ts:79]:
```suggestion
		presenceData.largeImageKey =
			document.querySelector<HTMLImageElement>(
				"#dynamic-height-slate > div > div > div.col-md-9 > div > div > div.profile-upper > div.rev-banking-background.profile-background > div > div.profile-avatar.col-md-3 > div > img"
			).src;
```

**Review by Slowlife01 — APPROVED** [review]

**Review by EncryptedDev — APPROVED** [review]

**Pull request merged**